plugins/logger.go

```diff
@@ -1,14 +1,14 @@
 package plugins
 
-import "github.com/odpf/meteor/internal/logger"
+import "github.com/odpf/salt/log"
 
 type Logger interface {
```

Comment:
We can reuse salt logger [interface](https://github.com/odpf/salt/blob/main/log/logger.go#L21) right? I mean I agree we should have our own interface but because salt is managed in odpf, we know what we are supporting so I guess we can reuse?

Reply:
Yes, Plan to remove this in the next PR.